src/components/Input/index.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { letterSpacing, colors, size } from '../../utils'
+
+const { black, gray3 } = colors
+
+const Input = styled.input`
+  width: 100%;
+  padding: 1rem;
+  box-sizing: border-box;
+  border: 1px solid rgba(224, 224, 224, 0.5);
+  background: rgba(196, 196, 196, 0.05);
+  outline: none;
+  font-weight: 700;
+  font-size: ${size.xxs};
+  letter-spacing: ${letterSpacing.medium};
+  font-family: Karu;
+  ::placeholder,
+  ::-webkit-input-placeholder {
+    color: ${black};
+  }
+  :focus {
+    ::placeholder {
+      color: transparent;
+    }
+  }
+`
+const Label = styled.label`
+  width: 100%;
+  text-transform: uppercase;
+  font-family: Karu;
+  font-weight: 550;
+  font-size: ${size.xxs};
+  letter-spacing: ${letterSpacing.medium};
+  color: ${gray3};
+`
+const Textarea = styled.textarea`
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 1rem;
+  border: 1px solid rgba(224, 224, 224, 0.5);
+  resize: none;
+  background: rgba(196, 196, 196, 0.05);
+  outline: none;
+  font-weight: 550;
+  font-size: ${size.xxs};
+  letter-spacing: ${letterSpacing.medium};
+  font-family: Karu;
+  ::placeholder,
+  ::-webkit-input-placeholder {
+    color: ${black};
+  }
+  :focus {
+    ::placeholder {
+      color: transparent;
+    }
+  }
+`
+
+const InputContainer = styled.div`
+  width: 45%;
+  margin: 10px 0;
+`
+const TextareaContainer = styled.div`
+  width: 100%;
+  height: 230px;
+  margin: 10px 0;
+  @media (max-width: 800px) {
+    height: 180px;
+  }
+`
+
+const CustomInput = ({ big, name, label, placeholder, handleChange }) =>
```

Comment:
Please rename it to onChange, the handler is the function and change is an event

Reply:
And the other namings are like so descriptive...
rename the props

Reply:
Instead of big, make it text area, even better? export the TextArea type apart from the 1 line Input, or at least rename into better props

Reply:
I think you are confused between onChange and handleChange see this: 
https://css-tricks.com/working-with-events-in-react/
https://reactjs.org/docs/forms.html
and you alright I'm passing the function not the event so it's named should handleChange

Reply:
I should rename big to textarea , good point

Reply:
Rename the handle click to onClick.
in the base html elements you type you type `onClick` or `handleClick` to give an element an event listener